csi/controller.go

```diff
@@ -610,6 +610,14 @@ func (s *OsdCsiServer) CreateVolume(
 		if err != nil {
 			return nil, err
 		}
+
+		_, err = volumes.Update(ctx, &api.SdkVolumeUpdateRequest{
```

Comment:
volumes.Clone calls driver.Snapshot, which removes `pvc/namespace` labels to avoid misuse. Another possible fix is to change driver.Snapshot to take additional pvc/namespace parameters, but this fix is far less invasive for a patch.